.github/workflows/benchmark.yml

```diff
@@ -35,6 +35,5 @@ jobs:
           external-data-json-path: ./benchmarks/data.json
           github-token: ${{ secrets.GITHUB_TOKEN }}
           gh-pages-branch: benchmarks
-          auto-push: true
```

Comment:
This is the root of the error. We can't rely on `external-data-json-path`, and `auto-push` at the same time.

> auto-push must be false when external-data-json-path is set since this action reads/writes the given JSON file and never pushes to remote

Also, auto-pushing seems like it'd be in violation of the guidance in https://github.com/open-telemetry/community/blob/main/docs/using-github-extensions.md#using-github-apps-extensions-and-bots